systemd/journal.py

```diff
@@ -587,7 +596,7 @@ def emit(self, record):
             self.handleError(record)
 
     @staticmethod
-    def mapPriority(levelno):
+    def map_priority(levelno):
```

Comment:
Yes, I agree that `map_priority` is a much better name. This part of the code is very old, it even predates the time when python-systemd was in the systemd repo. Hence the style that is a bit strange.

But I think we must keep backwards compatibility here. Can you add `mapPriority = map_priority` or something like that?